Web3Swift/Account/Account.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Created by Timofey on 3/19/18.
+//
+
+import Foundation
+
+//An account that is associated with some private key on the network
+public protocol Account {
+
+    /**
+    TODO: I am not sure whether the Account should promise to return its balance in wei
```

Comment:
@biboran  Probably we should create some sort of value object for `Wei` that can be easily converted to `Eth`, `GWei` and other units like [here](https://web3js.readthedocs.io/en/1.0/web3-utils.html#id79)